src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,72 @@
+import { ClassNames, css } from '@emotion/core';
+import * as React from 'react';
+import { Tab } from 'react-tabs';
+import { useTheme } from '../theme';
+
+const StyledTab: React.FunctionComponent<{ disabled?: boolean }> & { tabsRole: string } = props => {
+  const theme = useTheme();
+  const styles = tabStyles(theme.tabs);
+  return (
+    <ClassNames>
+      {({ css: getClassName }) => (
+        <Tab
+          {...props}
+          className={getClassName(styles.tabStyle)}
+          disabledClassName={getClassName(styles.disabledTabStyle)}
+          selectedClassName={getClassName(styles.selectedTabStyle)}
+        >
+          {props.children}
+        </Tab>
+      )}
+    </ClassNames>
+  );
+};
+
+StyledTab.tabsRole = 'Tab';
+
+const tabStyles = (tabTheme: { fg: string; bg: string }) => {
+  return {
+    tabStyle: css({
```

Comment:
You don't need `css` here.
A plain object literal is more than enough :)

Reply:
If TS complains, make sure to annotate return.
```tsx
const tabStyles = (tabTheme: { fg: string; bg: string }): Dictionary<IBoxCSS> => {
```
should do the trick for you

Reply:
Done!